ssr-server.js

```diff
@@ -79,7 +82,30 @@ async function preprender(req, res, next, data = {orbit: []}) {
 }
 
 app.get('/', preprender);
-app.get('/search', preprender);
+app.get('/search', async function(req, res, next) {
+  let searchTerm = req.query['search-term'];
+  if (searchTerm) {
+    res.redirect(`/search/${searchTerm}`);
+  } else {
+    await preprender(req, res, next);
+  }
+});
+
+app.get('/search-by-coordinates', async function(req, res, next) {
+  try {
+    let ip = req.clientIp;
+    let geoData = geoip.lookup(ip);
+
+    if (geoData) {
+      let [ lat, lon ] = geoData.ll;
+      res.redirect(`/search/${lat},${lon}`);
+    } else {
+      res.redirect(`/search/0,0`);
```

Comment:
Can we use values that are out of range, e.g. `200,200` or so?

Reply:
0,0 is also an impossible value due the precision of coordinates; but you're not right, it's not semantically correct. I'll change it to a better value.

Reply:
Correction: you're ~not~ right